debian/changelog

```diff
@@ -1,8 +1,26 @@
-cloud-init (22.1-1-gb3d9acdd-0ubuntu1~22.04.2) UNRELEASED; urgency=medium
+cloud-init (22.1-14-g2e17a0d6-0ubuntu1~22.04.1) jammy; urgency=medium
 
   * debian/apport-launcher.py: Fix string and whitespace formatting
+  * New upstream snapshot.
+    - check for existing symlink while force creating symlink (#1281)
+      [Shreenidhi Shedi]
+    - Do not silently ignore integer uid (#1280) (LP: #1875772)
+    - tests: create a IPv4/IPv6 VPC in Ec2 integration tests (#1291)
+    - Integration test fix ppa  (#1296)
+    - tests: on official EC2. cloud-id actually startswith aws not ec2 (#1289)
+    - test_ppa_source: accept both http and https URLs (#1292)
+      [Paride Legovini]
+    - Fix apt test on azure
+    - add "lkundrak" as contributor [Lubomir Rintel]
+    - Holmanb/integration test fix ppa (#1287)
+    - Include missing subcommand in manpage (#1279)
+    - Clean up artifacts from pytest, packaging, release with make clean
+      (#1277)
+    - sources/azure: ensure retries on IMDS request failure (#1271)
+      [Chris Patterson]
+    - sources/azure: removed unused savable PPS paths (#1268) [Chris Patterson]
```

Comment:
lintian gave a new warning related to this line from my sbuild cmd:

```bash
 sbuild --resolve-alternatives --dist=jammy --arch=amd64  --arch-all ../out/cloud-init_22.1-1-gb3d9acdd-0ubuntu1~22.04.2.dsc
...
W: cloud-init: spelling-error-in-changelog savable saveable
```

We need to fix this spelling so we don't end up introducing more lintian warnings than previous releases.

Reply:
This applies the the other ubuntu/<series> branches too